sdk/python/kfp/components/pipeline_task.py

```diff
@@ -401,8 +401,21 @@ def add_node_selector_constraint(self, accelerator: str) -> 'PipelineTask':
         """Sets accelerator type to use when executing this task.
 
         Args:
-            value: The name of the accelerator. Available values include
-                ``'NVIDIA_TESLA_K80'`` and ``'TPU_V3'``.
+            value: The name of the accelerator, such as ``'NVIDIA_TESLA_K80'`` and ``'TPU_V3'``.
+
+        Returns:
+            Self return to allow chained setting calls.
+        """
+        warnings.warn(
+            f'{self.add_node_selector_constraint.__name__!r} is deprecated. Please use {self.set_accelerator_type.__name__!r} instead.',
+            category=DeprecationWarning)
+        return self.set_accelerator_type(accelerator)
+
+    def set_accelerator_type(self, accelerator: str) -> 'PipelineTask':
+        """Sets accelerator type to use when executing this task.
+
+        Args:
+            value: The name of the accelerator, such as ``'NVIDIA_TESLA_K80'`` and ``'TPU_V3'``.
```

Comment:
For KFP users, they should specify the vendor of the accelerator. Valid values generally vary across platfoms, but the popular ones are: `nvidia.com/gpu`, `amd.com/gpu`, `cloud-tpus.google.com/v2`, `cloud-tpus.google.com/v3`.

I propose to mention at least `nvidia.com/gpu` and `cloud-tpus.google.com/v3` as OSS counterparts to `NVIDIA_TESLA_K80` and `TPU_V3`.

Reply:
Good thinking! Done.